catalog/core/catalog-core-solr/src/main/java/ddf/catalog/source/solr/DynamicSchemaResolver.java

```diff
@@ -237,7 +247,7 @@ public void addFields(Metacard metacard, SolrInputDocument solrInputDocument)
         List<Serializable> attributeValues = metacard.getAttribute(ad.getName()).getValues();
 
         if (attributeValues != null
-            && attributeValues.size() > 0
+            && !attributeValues.isEmpty()
```

Comment:
✏️  First 2 conditions can be replaced with `CollectionUtils.isEmpty(attributeValues)`